### PR TITLE
First pass at dukes tutorials

### PR DESCRIPTION
**Pull request by ggam**
Two concerns:
- I've used "Jakarta REST" in some places. I read somewhere that name was valid, but I'm not sure about that. Can anyone confirm?
- I've left "Bean Validation" without the Jakarta prefix in a few places to reduce verbosity. Is this acceptable?

### CODE REVIEW

**Comment by bshannon** [comment]
> Two concerns:
> 
>     * I've used "Jakarta REST" in some places. I read somewhere that name was valid, but I'm not sure about that. Can anyone confirm?

I don't believe the project team accepted that name, but feel free to double check with them.

>     * I've left "Bean Validation" without the Jakarta prefix in a few places to reduce verbosity. Is this acceptable?

Yes.

File: src/main/jbake/content/dukes-bookstore.adoc [hunk 1]
```diff
@@ -14,9 +14,9 @@ Duke's Bookstore Case Study Example
 
 
 The Duke's Bookstore example is a simple e-commerce application that
-illustrates some of the more advanced features of JavaServer Faces
-technology in combination with Contexts and Dependency Injection for
-Jakarta EE (CDI), enterprise beans, and the Java Persistence API. Users can
+illustrates some of the more advanced features of Jakarta Faces
```
Comment (bshannon) [inline src/main/jbake/content/dukes-bookstore.adoc]: Jakarta Server Faces

(I don't need to keep marking this, do I?  :-))

File: src/main/jbake/content/dukes-bookstore.adoc [hunk 2]
```diff
@@ -14,9 +14,9 @@ Duke's Bookstore Case Study Example
...
+technology in combination with Jakarta Dependency Injection for
```
Comment (bshannon) [inline src/main/jbake/content/dukes-bookstore.adoc]: Jakarta Contexts and Dependency Injection

File: src/main/jbake/content/dukes-forest001.adoc [hunk 3]
```diff
@@ -16,13 +16,13 @@ Duke's Forest is a simple e-commerce application that contains several
...
 
-* JavaServer Faces technology, including Ajax
-* Contexts and Dependency Injection for Jakarta EE (CDI)
-* Java API for RESTful Web Services (JAX-RS)
-* Java Persistence API (JPA)
-* Java API for JavaBeans Validation (Bean Validation)
-* Enterprise JavaBeans (EJB) technology
-* Java Message Service (JMS)
+* Jakarta Faces technology, including Ajax
+* Jakarta Dependency Injection (CDI)
```
Comment (bshannon) [inline src/main/jbake/content/dukes-forest001.adoc]: Jakarta Contexts and Dependency Injection

File: src/main/jbake/content/dukes-tutoring001.adoc [hunk 4]
```diff
@@ -67,7 +67,7 @@ methods
 
 ** A WebSocket server endpoint that automatically publishes the status
 of students to client endpoints
-* Contexts and Dependency Injection
+* Jakarta Dependency Injection
```
Comment (bshannon) [inline src/main/jbake/content/dukes-tutoring001.adoc]: Jakarta Contexts and Dependency Injection

File: src/main/jbake/content/dukes-forest001.adoc (hunk 3 above)
Comment (ggam) [reply src/main/jbake/content/dukes-forest001.adoc]: @antoinesd @starksm64 as CDI spec leads, is it valid to use Jakarta Dependency Injection as short name?

Comment (bshannon) [reply src/main/jbake/content/dukes-forest001.adoc]: [Jakarta Dependency Injection](https://jakarta.ee/specifications/dependency-injection/1.0/) is a different spec than [Jakarta Contexts and Dependency Injection](https://jakarta.ee/specifications/cdi/2.0/).  CDI builds on DI.

Comment (ggam) [reply src/main/jbake/content/dukes-forest001.adoc]: Ok, tha's why I was so sure I had read Jakarta Dependency Injection somewhere... will take care of it.

File: src/main/jbake/content/dukes-bookstore.adoc [hunk 5]
```diff
@@ -14,9 +14,9 @@ Duke's Bookstore Case Study Example
 
 
 The Duke's Bookstore example is a simple e-commerce application that
-illustrates some of the more advanced features of JavaServer Faces
-technology in combination with Contexts and Dependency Injection for
-Jakarta EE (CDI), enterprise beans, and the Java Persistence API. Users can
+illustrates some of the more advanced features of Jakarta Faces
+technology in combination with Jakarta Contexts and Dependency Injection for
```
Comment (bshannon) [inline src/main/jbake/content/dukes-bookstore.adoc]: Remove the "for".

**Pull request merged**